test/verify/check-machines

```diff
@@ -262,6 +262,14 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         if args["logfile"] is not None:
             b.wait(lambda: "NMI received" in self.machine.execute("cat {0}".format(args["logfile"])))
 
+        # pause
+        b.click("#vm-subVmTest1-pause")
+        b.wait_in_text("#vm-subVmTest1-state", "paused")
```

Comment:
Just testing the UI is too shallow IMHO -- please also do a wait(virsh) check that the VM *actually* gets paused/resumed. I know that the selenium test didn't do that, but it's both easy to add and also too easy to forget to do it later :-)

Reply:
Now the UI responsiveness to virsh is tested. Few lines above.